sqlserver/datadog_checks/sqlserver/schemas.py

```diff
@@ -204,7 +204,7 @@ def _fetch_for_databases(self):
                                 db_name, e
                             )
                         )
-                        return
+                        break
```

Comment:
This works, but to be safer, should we put the for loop in a `try...finally` block?

Reply:
yep, thought about it but as everything is in a "try except" inside of a for loop decided to go for the `break`. 

But , yeah , will change, safer if someone modifies something later.